integration/walletextension/wallet_extension_test.go

```diff
@@ -546,19 +557,27 @@ func makeEthJSONReqAsJSON(t *testing.T, walletExtensionAddr string, method strin
 	return respBodyJSON
 }
 
+func generateAndSubmitWalletViewingKey(t *testing.T, walletExtensionAddr string, accountPrivateKey *ecdsa.PrivateKey) {
```

Comment:
@BedrockSquirrel I think we can kill off this method? It only has one usage, and only wraps another call.